updater/src/Updater/Generate/Changelog.purs

```diff
@@ -0,0 +1,63 @@
+module Updater.Generate.Changelog where
+
+import Prelude
+
+import Affjax as AX
+import Affjax.ResponseFormat as RF
+import Data.Codec (decode)
+import Data.Codec.Argonaut (JsonCodec, array, printJsonDecodeError)
+import Data.Codec.Argonaut as CA
+import Data.Codec.Argonaut.Record as CAR
+import Data.Either (Either(..))
+import Data.Foldable (foldl)
+import Data.String.CodeUnits (takeWhile)
+import Data.String.Common (joinWith, trim)
+import Effect.Aff (Aff, error, throwError)
+import Node.Encoding (Encoding(..))
+import Node.FS.Aff as FSA
+
+type ReleaseInfo =
+  { tag_name :: String
+  , html_url :: String
+  , body :: String
+  , published_at :: String
+  }
+
+releaseCodec :: JsonCodec ReleaseInfo
+releaseCodec =
+  CAR.object "ReleaseInfo" $
+    { tag_name: CA.string
+    , html_url: CA.string
+    , body: CA.string
+    , published_at: CA.string
+    }
+
+appendReleaseInfoToChangelog :: forall r. { owner :: String, repo :: String | r } -> Aff Unit
+appendReleaseInfoToChangelog gh = do
+  -- For example
+  -- https://api.github.com/repos/purescript-contrib/purescript-http-methods/releases
+  let url = "https://api.github.com/repos/" <> gh.owner <> "/" <> gh.repo <> "/releases"
```

Comment:
Do you know how many releases are returned? Is it all releases, or the first `n`?

Reply:
I just checked using `purescript/purescript`, which has 89 releaess. Looks like it's just the first 30.

Reply:
In that case we either need to set a high upper limit on the number of releases included (for example, by passing a parameter to the API), or we need to page them. @mhmdanas is working on something similar in https://github.com/purescript-contrib/setup-purescript/pull/11 but I don't think it's complete yet.

Reply:
Looking at the docs, I think [we'll need to page them](https://docs.github.com/en/rest/reference/repos#list-releases)

Reply:
Ok, so maximum per page is 100. Anything beyond that and we'll need to page them. I don't think we'll run into that in our repos, but I've updated the code to handle that even if there are more.